modules/host/obligations.go

```diff
@@ -0,0 +1,264 @@
+package host
+
+import (
+	"os"
+	"sync"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+const (
+	// resubmissionTimeout is the number of blocks that the host will wait
+	// before trying to resubmit a transaction to the blockchain.
+	resubmissionTimeout = 2
+
+	// confirmationRequirement is the number of blocks that the host is going
+	// to wait before assuming that a storage proof has successfully been
+	// confirmed by the blockchain.
+	confirmationRequirement = 12
+
+	// Booleans to indicate that a contract obligation has been successful or
+	// unsuccessful.
+	obligationSucceeded = true
+	obligationFailed    = false
```

Comment:
alternatively, define separate functions for removing successful and unsuccessful obligations